src/gameplay/room.js

```diff
@@ -37,6 +37,7 @@ class Room {
     this.ranked = ranked_;
     this.gamesRequiredForRanked = 0;
     this.provisionalGamesRequired = 20;
+    this.timer = 25 * 60;
```

Comment:
This should probably be configurable (i.e. passed in the constructor), but for now it's ok to keep here during testing :)